mozjs-sys/mozjs/build/moz.configure/toolchain.configure

```diff
@@ -1779,11 +1778,11 @@ def select_linker_tmpl(host_or_target):
             # ensure consistent output.
             env["LC_ALL"] = "C"
             retcode, stdout, stderr = get_cmd_output(*cmd, env=env)
-            if retcode == 1 and "Logging ld64 options" in stderr:
+            if retcode == 1 and ("Logging ld64 options" in stderr or "PROGRAM:ld PROJECT:dyld" in stderr):
                 kind = "ld64"
 
-            elif retcode != 0:
-                return None
+            elif retcode != 0 and version_check == "-Wl,--version":
+                return try_linker(linker, "-Wl,-v")
```

Comment:
Any reason you aren't exiting early here like in the old version ala:

```
elif retcode != 0
    if version_check == "-Wl,--version":
        try_linker(linker, "-Wl,-v")
   else:
        return None
```

Reply:
Hm, we could, but this way we fallback to -v for all linkers.

Reply:
I did revert this to keep the change simpler and we could reintroduce this if needed.